client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -456,133 +346,172 @@ function TitanMailMeta( { item, isRenewal } ) {
+						  )
+				);
 			}
-			break;
+			return { title, description };
+		}
 		case 'plan':
-			modalCopy.title = translate( 'You are about to remove your plan from the cart' );
-			modalCopy.description = createUserAndSiteBeforeTransaction
-				? 'When you press Continue, we will remove your plan from the cart.'
-				: translate(
-						'When you press Continue, we will remove your plan from the cart and your site will continue to run with its current plan. We will then take you back to your site.'
-				  );
-			break;
+			return {
+				title: String( translate( 'You are about to remove your plan from the cart' ) ),
+				description: String(
+					createUserAndSiteBeforeTransaction
+						? translate( 'When you press Continue, we will remove your plan from the cart.' )
+						: translate(
+								'When you press Continue, we will remove your plan from the cart and your site will continue to run with its current plan. We will then take you back to your site.'
+						  )
+				),
+			};
 		case 'domain':
-			modalCopy.title = translate( 'You are about to remove your domain from the cart' );
-			modalCopy.description = translate(
-				'When you press Continue, we will remove your domain from the cart and you will have no claim for the domain name you picked.'
-			);
-			break;
+			return {
+				title: String( translate( 'You are about to remove your domain from the cart' ) ),
+				description: String(
+					translate(
+						'When you press Continue, we will remove your domain from the cart and you will have no claim for the domain name you picked.'
+					)
+				),
+			};
 		case 'coupon':
-			modalCopy.title = translate( 'You are about to remove your coupon from the cart' );
-			modalCopy.description = translate(
-				'When you press Continue, we will need you to confirm your payment details.'
-			);
-			break;
+			return {
+				title: String( translate( 'You are about to remove your coupon from the cart' ) ),
+				description: String(
+					translate( 'When you press Continue, we will need you to confirm your payment details.' )
+				),
+			};
 		default:
-			modalCopy.title = translate( 'You are about to remove your product from the cart' );
-			modalCopy.description = createUserAndSiteBeforeTransaction
-				? 'When you press Continue, we will remove your product from the cart.'
-				: translate(
-						'When you press Continue, we will remove your product from the cart and your site will continue to run without it.'
-				  );
+			return {
+				title: String( translate( 'You are about to remove your product from the cart' ) ),
+				description: String(
+					createUserAndSiteBeforeTransaction
+						? translate( 'When you press Continue, we will remove your product from the cart.' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'When you press Continue, we will remove your product from the cart and your site will continue to run without it.' )` *ES Score: 6*

Reply:
I don't know why these strings were not translated in the first place when they were added, but this is different because the site does not exist yet.